README.md

```diff
@@ -97,15 +97,13 @@ the conf file will be mounted into docker container at runtime to PGSERVICEFILE
 
 To active https with LetsEncrypt just follow the following instructions:
 
-- move `config/_nginx/django_ssl.conf` to `config/nginx/django_ssl.conf`
-- check the domain name in the `.env` file and in `config/nginx/django_ssl.conf`
-- run: `docker pull certbot/certbot`
+- uncomment ssl section within `config/nginx/django.conf`
+- update `WEBGIS_PUBLIC_HOSTNAME` environment variable within the `.env` file
 - launch `./run_certbot.sh`
-- activate 301 redirect into `config/nginx/django.conf`
 - restart compose
```

Comment:
ambiguous, rewrite better to indicate the exact terminal command needed to restart compose

Reply:
> terminal command needed to restart compose
>
> `docker compose down && docker compose up -d`

by chance, does this do the same? `docker compose up -d --force-recreate`

Ref: [Compose UP options](https://docs.docker.com/engine/reference/commandline/compose_up/#options)